src/applications/static-pages/download-1095b/mocks/endpoints/feature-toggles/index.js

```diff
@@ -0,0 +1,66 @@
+const { snakeCase } = require('lodash');
```

Comment:
This whole file borrowed from Profile and condensed for specific usage in this app.